oeps/conventional-commits.rst

```diff
@@ -0,0 +1,77 @@
+####################
+Conventional Commits
+####################
+
+Abstract
+********
+
+Commits should be clearly labeled for their effect on consumers of the repository.  To do this, we adopt the `Conventional Commits`_ guidelines, which detail structured commit messages that clarify the impact of each commit.
+
+This is part of our Change Transparency initiative.
+
+Spec
+****
+
+We are adopting the `Conventional Commits`_ spec, with our own adjustments.
+
+.. note:: Many people are familiar with the `Angular commit message format`_, which uses conventional commits and inspired the Conventional Commits spec.  We are not adopting the Angular rules.
+
+Commit messages have these parts::
+
+    <type>: <subject>
+
+    <body>
+
+    <footer>
+
+Type
+====
+
+We use these commit types:
```

Comment:
What do we think about adding an `api` commit type?  Are APIs that are built for Extensions designated as `feat` or do we introduce a new category for this?

Reply:
"api" is the kind of thing that would be part of the scope if we were using them.  Perhaps scopes is the solution to the "user-visible" aspect also?